admin/mirage/config.js

```diff
@@ -60,6 +60,12 @@ export default function () {
     return adminMember.update({ role });
   });
 
+  this.put('/admin/admin-members/:id/deactivate', (schema, request) => {
+    const id = request.params.id;
+    const adminMember = schema.adminMembers.findBy({ id });
+    return adminMember;
```

Comment:
**suggestion**
Il faudrait plutôt supprimer le membre plutôt que de le renvoyer ici non ?

Reply:
Yes, je peux essayer de faire ça, ou de ne rien faire car la requête renvoie une 204 maintenant ?
Dans la member action, l'agent est supprimé grâce à 
`   after() {
      this.unloadRecord();
    },`

Reply:
Oui c'est ça